examples/vision/randaugment.py

```diff
@@ -0,0 +1,320 @@
+[`cifar10_corrupted/saturate_5`](https://www.tensorflow.org/datasets/catalog/cifar10_corrupted#cifar10_corruptedsaturate_5). The images from this configuration look like so:
+
+![](https://storage.googleapis.com/tfds-data/visualization/fig/cifar10_corrupted-saturate_5-1.0.0.png)
+
+For the sake of reproducibility, we serialize the initial random weights of our shallow
+network. 
+"""
+
+initial_model = get_training_model()
+initial_model.save_weights("initial_weights.h5")
+
+"""
+#1. Train model with RandAugment
+"""
+
+rand_aug_model = get_training_model()
+rand_aug_model.load_weights("initial_weights.h5")
+rand_aug_model.compile(
+    loss="sparse_categorical_crossentropy", optimizer="adam", metrics=["accuracy"]
+)
+rand_aug_model.fit(train_ds_rand, validation_data=test_ds, epochs=EPOCHS)
+_, test_acc = rand_aug_model.evaluate(test_ds)
+print("Test accuracy: {:.2f}%".format(test_acc * 100))
+
+"""
+# 2. Train model with `simple_aug`
+"""
+
+simple_aug_model = get_training_model()
+simple_aug_model.load_weights("initial_weights.h5")
+simple_aug_model.compile(
+    loss="sparse_categorical_crossentropy", optimizer="adam", metrics=["accuracy"]
+)
+simple_aug_model.fit(train_ds_simple, validation_data=test_ds, epochs=EPOCHS)
+_, test_acc = simple_aug_model.evaluate(test_ds)
+print("Test accuracy: {:.2f}%".format(test_acc * 100))
+
+"""
+## Load the CIFAR-10-C dataset and evaluate performance
+"""
+
+# Load and prepare the CIFAR-10-C dataset
+# (If it's not already downloaded, it takes ~10 minutes of time to download)
+cifar_10_c = tfds.load("cifar10_corrupted/saturate_5", split="test", as_supervised=True)
+cifar_10_c = cifar_10_c.batch(BATCH_SIZE)
+
+# Evaluate `rand_aug_model`
+_, test_acc = rand_aug_model.evaluate(cifar_10_c, verbose=0)
+print(
+    "Accuracy with RandAugment on CIFAR-10-C (saturate_5): {:.2f}%".format(
+        test_acc * 100
+    )
+)
+
+# Evaluate `simple_aug_model`
+_, test_acc = simple_aug_model.evaluate(cifar_10_c, verbose=0)
+print(
+    "Accuracy with simple_aug on CIFAR-10-C (saturate_5): {:.2f}%".format(
+        test_acc * 100
+    )
+)
+
+"""
+For the purpose of this example, we trained the models for only a single epoch. In my 
+experiments, I found that with RandAugment the model performs way better (76.64%) than
+the model trained with `simple_aug` (64.80%) on the CIFAR-10-C dataset. Additionally, I 
+found that RandAugment helped stabilize the training. You can my experimentation notebook
+[here](https://nbviewer.jupyter.org/github/sayakpaul/Keras-Examples-RandAugment/blob/main/RandAugment.ipynb). 
+
+As we can see from the notebook, at the expense of increased training time with RandAugment,
+we are able to carve out far better performance on the CIFAR-10-C dataset. You can
+experiment on the other corruption and perturbation settings that come with the
+run the same CIFAR-10-C dataset and see if RandAugment helps.
+
+Readers are encouraged to experiment with the different values of `n` and `m` in the
```

Comment:
Above you say "you can", here you can also say "you can also" instead of "readers are encouraged..."